Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN go mod download
 COPY cmd/main.go cmd/main.go
 COPY api/ api/
 COPY internal/ internal/
+COPY providers/ providers/
```

Comment:
Is it still needed? As I see, the image does not contain the providers anymore, it's mounted as a configmap now.

Reply:
Yea, needed to run build and units, this is for 1st stage of docker file